### ISSUE TITLE
Column 'login' in table 'sec_user' is of a type that is invalid for use as a key column in an index.

### ISSUE THREAD

**Issue by chrisknoll** [OHDSI/WebAPI#1578]
Error in SqlServer migration V2.8.0.20200427161830__modify_user_login.sql.

The statements:
```
ALTER TABLE ohdsi.sec_user ALTER COLUMN login VARCHAR(MAX);

ALTER TABLE ohdsi.sec_user ADD CONSTRAINT sec_user_login_unique UNIQUE (login);
```

fails because the login is a VARCHAR(MAX).  Why would we allow a login id to be a 2GB value?

Would varchar(500) be more reasonable?

**Comment by kkadapa** [comment]
I am getting the same error with the latest WebAPI v2.7.8 release. Is there a workaround for this error? 

When I deploy the WAR to Tomcat i constantly get the below error...

`FAIL - Application at context path [/WebAPI] could not be started
FAIL - Encountered exception [org.apache.catalina.LifecycleException: Failed to start component [StandardEngine[Catalina].StandardHost[localhost].StandardContext[/WebAPI]]]`

**Comment by chrisknoll** [comment]
master has a fix applied, making that column 1024.

**Comment by mintmountain89** [comment]
> I am getting the same error with the latest WebAPI v2.7.8 release. Is there a workaround for this error?

@chrisknoll  I'm getting this error in the tomcat logs too. Is this the correct version: https://github.com/OHDSI/WebAPI/releases/tag/v2.7.8

**Comment by chrisknoll** [comment]
No, in 2.7.8 and 2.7.9, that column is still varchar(max).  @wivern, @ssuvorov-fls , could you apply this fix over to the 2.7 release?  We have to be very careful about the 2.7 and 2.8 versions executing the same migration steps, but it's probably fine if both versions attempt to update the column type.

@mintmountain89 : we're attempting to get a 2.8 release finished by end of year, so, you may be able to get the fix for this via a 2.8 release instead of waiting for a 2.7.10 release.

**Comment by mintmountain89** [comment]
nice 👍 , thank you.
